src/controller/python/test/test_scripts/mobile-device-test.py

```diff
@@ -61,13 +61,13 @@ def main():
                                    nodeid=1),
               "Failed to finish key exchange")
 
-    logger.info("Testing network commissioning")
-    FailIfNot(test.TestNetworkCommissioning(nodeid=1,
-                                            endpoint=ENDPOINT_ID,
-                                            group=GROUP_ID,
-                                            dataset=TEST_THREAD_NETWORK_DATASET_TLV,
-                                            network_id=TEST_THREAD_NETWORK_ID),
-              "Failed to finish network commissioning")
+    # logger.info("Testing network commissioning")
```

Comment:
Why commented?

Reply:
Sorry, mistakenly commented for local tests